README.adoc

```diff
@@ -11,29 +11,29 @@
 //
 :page-layout: guide
 // INSTRUCTION: The project id is the part of the git repository after the guide- and must be specified
-:projectid: github repo name without the guide- prefix
```

Comment:
Keep the original statement "github repo name without the guide- prefix" as comment.